Source/WebCore/platform/graphics/NativeImage.h

```diff
@@ -55,9 +55,11 @@ class NativeImage final : public RenderingResource {
     bool hasAlpha() const;
     std::optional<Color> singlePixelSolidColor() const;
     WEBCORE_EXPORT DestinationColorSpace colorSpace() const;
+    WEBCORE_EXPORT bool hasHDRContent() const;
     WEBCORE_EXPORT Headroom headroom() const;
 
     void draw(GraphicsContext&, const FloatRect& destRect, const FloatRect& srcRect, ImagePaintingOptions);
+    void drawToneMapping(GraphicsContext&, const FloatRect& destinationRect, const FloatRect& sourceRect, ImagePaintingOptions);
```

Comment:
Why not just add a flag to `ImagePaintingOptions`? If you keep this, call it `drawWithToneMapping()` or `drawToneMapped()`.

Reply:
The name was changed to `drawWithToneMapping()`.  Having a flag in `ImagePaintingOptions` seems a little bit complicated.